openshift/build.yaml

```diff
@@ -9,7 +9,7 @@ parameters:
   - name: GIT_URL
     value: https://github.com/bcgov/wps-web.git
   - name: GIT_REF
-    value: master
+    value: nkuruba-master
```

Comment:
Isn't there a better way to point the template to the branch we want to use? Hard coding to a particular branch doesn't seem like a good solution.

Reply:
The branch name has been updated to master.